### PR TITLE
added some translation

### PR DESCRIPTION
**Pull request by AlwaysReVaule**
two sentences in  Chinese.

### CODE REVIEW

File: files/zh-cn/learn/tools_and_testing/client-side_javascript_frameworks/vue_styling/index.html [hunk 1]
```diff
@@ -302,9 +302,9 @@ <h3 id="Adding_CSS_classes_in_Vue">Adding CSS classes in Vue</h3>
 
 <pre class="brush: html notranslate">&lt;ul aria-labelledby="list-summary" class="stack-large"&gt;</pre>
 
-<h2 id="Adding_scoped_styles">Adding scoped styles</h2>
+<h2 id="Adding_scoped_styles">增加局部的样式/h2>
```
Comment (yin1999) [inline files/zh-cn/learn/tools_and_testing/client-side_javascript_frameworks/vue_styling/index.html:305]:
```suggestion
<h2 id="Adding_scoped_styles">增加局部样式/h2>
```

File: files/zh-cn/learn/tools_and_testing/client-side_javascript_frameworks/vue_styling/index.html [hunk 2]
```diff
@@ -302,9 +302,9 @@ <h3 id="Adding_CSS_classes_in_Vue">Adding CSS classes in Vue</h3>
...
 
-<p>The last component we want to style is our <code>ToDoItem</code> component. To keep the style definitions close to the component we can add a <code>&lt;style&gt;</code> element inside it. However, if these styles alter things outside of this component, it could be challenging to track down the styles responsible, and fix the problem. This is where the <code>scoped</code> attribute can be useful — this attaches a unique HTML <code>data</code> attribute selector to all of your styles, preventing them from colliding globally.</p>
+<p>我们最后想要修改样式的组件是我们的 <code>ToDoItem</code> 组件。为了保持样式定义更贴近组件，我们可以在里面增加一个<code>&lt;style&gt;</code> 元素。然而，如果这些样式的改变发生在组件外部，那么就可以追溯样式的责任并修复问题。这就是<code>scoped</code> 属性的作用所在- 这里附加了一个唯一的HTML <code>data</code>属性选择器给你所有的样式, 已避免全局冲突.</p>
```
Comment (yin1999) [inline files/zh-cn/learn/tools_and_testing/client-side_javascript_frameworks/vue_styling/index.html:307]: 这段话的翻译不够准确，请继续调整：

```suggestion
<p>我们最后想要修改样式的组件是我们的 <code>ToDoItem</code> 组件。为了保持样式定义更贴近组件，我们可以在里面增加一个 <code>&lt;style&gt;</code> 元素。然而，如果这些样式的改变发生在组件外部，那么追溯存在问题的样式并完成修复将变得较为困难。这就是 <code>scoped</code> 属性的作用所在——这里附加了一个唯一的 HTML <code>data</code> 属性选择器给你所有的样式，以避免全局冲突。</p>
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/learn/tools_and_testing/client-side_javascript_frameworks/vue_styling/index.html (hunk 1 above)
Comment (AlwaysReVaule) [reply files/zh-cn/learn/tools_and_testing/client-side_javascript_frameworks/vue_styling/index.html:305]: agree :)

File: files/zh-cn/learn/tools_and_testing/client-side_javascript_frameworks/vue_styling/index.html (hunk 2 above)
Comment (AlwaysReVaule) [reply files/zh-cn/learn/tools_and_testing/client-side_javascript_frameworks/vue_styling/index.html:307]: all right. :）

**Pull request closed without merging**